typos.toml

```diff
@@ -0,0 +1,42 @@
+[files]
+extend-exclude = [
+    # exclude git (sub)modules and generated content
+    "compiler/rustc_codegen_gcc",
+    "compiler/rustc_codegen_cranelift",
+    "compiler/rustc_baked_icu_data",
+]
+
+[default.extend-words]
+# Add exclusions here, lines should be like `x = "x"`, where `x` is excluded word.
+#
+# Also see docs: https://github.com/crate-ci/typos/blob/v1.28.2/docs/reference.md
```

Comment:
Based on the documentation these aren't exclusions, they're custom additions to the dictionary.

Reply:
Yes, but it is one of the ways to silence linter, so i wrote as `exclusions`, to simplify wording.